app/views/crystals/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with url: "/crystals/#{@crystal.id}", method: :patch, local: true do |form| %>
+  <%= form.label :name, "Name" %>
+  <%= form.text_field :name %>
```

Comment:
you can prefill this with the current name using value: @crystal.name if you like.

Reply:
oooh! thank you for the reminder! I just pushed it!